Simplenote/DeleteAccountRemote.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+class DeleteAccountRemote: Remote {
```

Comment:
WDYT about `AccountRemote` (and merge with `AccountVerificationRemote`), or `AccountDeletionRemote` (for consistency?)

Reply:
I like the idea of combining the account remotes :)